ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -706,13 +755,61 @@ def get_internet_exit_connections(self, internet_exit_policy: dict) -> list:
         """
         Return a list of connections (dicts) for the given internet_exit_policy.
 
-        These are useful for easy creation of connectivity-monitor, service-intersion connections, exit-groups, tunnels etc.
+        These are useful for easy creation of connectivity-monitor, service-insertion connections, exit-groups, tunnels etc.
         """
         policy_name = internet_exit_policy["name"]
+        policy_type = internet_exit_policy["type"]
+
+        if policy_type == "direct":
+            return self.get_direct_internet_exit_connections(internet_exit_policy)
+        elif policy_type == "zscaler":
+            return self.get_zscaler_internet_exit_connections(internet_exit_policy)
+        else:
+            raise AristaAvdError(f"Unsupported type '{policy_type}' found in cv_pathfinder_internet_exit[name={policy_name}].")
+
+    def get_direct_internet_exit_connections(self, internet_exit_policy: dict) -> list:
+        """
+        Return a list of connections (dicts) for the given internet_exit_policy of type direct.
+        """
+        if get(internet_exit_policy, "type") != "direct":
+            return []
+
+        connections = []
+
+        # Check if the policy has any local interface
+        for wan_interface in self.shared_utils.wan_interfaces:
+            wan_interface_internet_exit_policies = get(wan_interface, "cv_pathfinder_internet_exit.policies", default=[])
+            if get_item(wan_interface_internet_exit_policies, "name", internet_exit_policy["name"]) is None:
+                continue
+
+            if not wan_interface.get("peer_ip"):
+                raise AristaAvdMissingVariableError(
+                    f"{wan_interface['name']} peer_ip needs to be set. When using wan interface "
+                    "for direct type internet exit, peer_ip is used for nexthop, and connectivity monitoring."
+                )
+            connections.append(
+                {
+                    "type": "ethernet",
+                    "name": f"IE-{wan_interface['name']}",
+                    "monitor_name": f"IE-{wan_interface['name']}",
```

Comment:
Need to sanitize the name if it contians any `/` - let's try to reuse the logic for STUN so that we modify this centrally (`/` -> `_`)

Reply:
Not sure what you mean by centrally, added a sanitize function for now.

Reply:
Also didn't see much what should be added to the doc, take a look if what I added is enough.
If not what are you looking to be added.